src/content/en/updates/2017/06/supercharged-codesplit.md

```diff
@@ -0,0 +1,281 @@
+{# wf_tags: supercharged #}
+{# wf_featured_image: /web/updates/images/supercharged/poster.jpg #}
+{# wf_featured_snippet: “Code splitting” and “route-based chunking” are some of the new buzzwords in web development. But what do they mean and how can you implement them? #}
+
+
+# Supercharged Live Stream Blog: Code Splitting {: .page-title }
+
+{% include "web/_shared/contributors/surma.html" %}
+
+Note: As always – **this is not production-ready code**. The code makes a couple assumptions to make
+the code (and the video) easier to follow at the cost of generality. Our main goal is to convey
+concepts and demystify buzzwords.
+
+In our most recent Supercharged Live Stream we implemented code splitting and route-based chunking.
+With HTTP/2 and native ES6 modules, these techniques will become essential to enabling efficient
+loading and caching of script resources.
+
+<div class="video-wrapper-full-width">
+  <iframe class="devsite-embedded-youtube-video" data-video-id="4KVeNoN1aFM"
+            data-autohide="1" data-showinfo="0" frameborder="0" allowfullscreen>
+  </iframe>
+</div>
+
+### Miscellaneous tips & tricks in this episode
+
+- `asyncFunction().catch()` with `error.stack`: 9:55
+- Modules and `nomodule` attribute on `<script>` tags: 7:30
+- `promisify()` in Node 8: 17:20
+
+### TL;DR: {: .hide-from-toc }
+
+How to do code splitting via route-based chunking:
+
+1. Obtain a list of your entry points
+2. Extract the module dependencies of all these entry points
+3. Find shared dependencies between all entry points
+4. Bundle the shared dependencies
+5. Rewrite entry points
+
+### Code splitting vs route-based chunking
+
+> Time stamp: 1:50
+
+Code splitting and route-based chunking are closely related and are often used interchangably. This
+has caused some confusion. Let’s try to clear this up:
+
+* **Code splitting**: Code splitting is the process of *not* shipping one big bundle with all of
+your JavaScript to the client. Instead, the code is split into multiple bundles. One specific way of
+splitting your code is to use route-based chunking.
+* **Route-based chunking**: Route-based chunking creates bundles that are related your app’s routes.
+By analyzing your routes and their dependencies, we can make decision what modules go into which
+bundle.
+
+### Why do code splitting?
+
+#### Loose modules
+
+With natives ES6 modules, every JavaScript module can import its own dependencies. When the browser
+receives a module, all `import` statements will trigger additional fetches to get ahold of the
+modules that are necessary to run the code. However, all these modules can have dependencies of
+their own. The danger is that the browser ends up with a cascade of fetches that lasts for multiple
+round-trip times (RTT) before the code can finally be executed.
+
+**ADD IMAGE for loose modules**
+
+#### Bundling
+
+Inlining all your modules into one single bundle (“bundling”) will make sure the browser has all the
```

Comment:
Since 'bundling' is the name of your section I don't like that it's in parenthesis. Try this:

'Bundling, which is inlining modules into a single download, makes sure the browser...'